VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,51 @@
+# Video 3: What is AsyncAPI? 
+ 
+Hello everyone! Welcome!
+		
+This video will help you understand what AsyncAPI is and how to continue learning about this Open-Source project. 
+
+In this video, we'll talk about:
+ - What is AsyncAPI?
+ - How did AsyncAPI come about?
+ - What AsyncAPI brings 
+ - How AsyncAPI Works
+ 
+ Ready? Let's get started!
+
+## Definition of AsyncAPI
+
+AsyncAPI is an open-source specification that aims to make working with EDA as easy today to work with RESTful APIs. 
+
+The strategy is to continue improving, implementing resources that make it more complete and solid. This has led to the recent merger with the Linux Foundation last year. Hosting the project on a neutral foundation guarantees the long-term sustainability and success of the initiative. 
+
+Another important step the initiative took towards its consolidation was to partner with Postman at the end of 2020, a leading collaborative platform developing APIs and a reference in the sector worldwide. This partnership allows it to drive the growth of more and better tools to help engineers create and maintain asynchronous APIs while using their usual languages and working environments.
+
+AsyncAPI currently aims to accomplish three main goals:
+
+1. Unify all API specifications. No one does event-driven architecture alone. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs.
+
+2. Improve the performance of the specification to make it more and more seamless. Users should be able to create their first API in a matter of minutes without prior knowledge of AsyncAPI.
+
+3. Reach the first position among the most used API specifications. This will require a growth in the number of community members of around 400% over the next two years.
```

Comment:
`This will require a growth in the number of community members of around 400% over the next two years.` problem is that 1 year passed and in 1 year more this sentence will be outdated. Let's just write `This will require a growth in the number of community members by ensuring it is a community-driven project running under open governance` ?